internal/services/sentinel/sentinel_alert_rule_scheduled_resource_test.go

```diff
@@ -202,6 +202,10 @@ resource "azurerm_sentinel_alert_rule_scheduled" "test" {
     display_name_format  = "Suspicious activity was made by {{ComputerIP}}"
     severity_column_name = "Computer"
     tactics_column_name  = "Computer"
+    dynamic_properties {
+      name  = "AlertLink"
+      value = "dcount_ResourceId"
+    }
```

Comment:
should this not be dynamic_property so
```suggestion
    dynamic_property {
      name  = "AlertLink"
      value = "dcount_ResourceId"
    }
    dynamic_property {
      name  = "AlertLink"
      value = "dcount_ResourceId"
    }

```
?